__tests__/endpoints.test.js

```diff
@@ -13,31 +13,29 @@ describe('GET /api/topics', () => {
     test('Should return an array of topic objects, each of which should have a slug & description', () => {
         return request(app).get("/api/topics")
             .expect(200)
-            .then(({body}) => {
-                    expect(body.topics).toMatchObject(
-                        [{
-                              description: 'The man, the Mitch, the legend',
-                              slug: 'mitch'
-                            },
-                            {
-                              description: 'Not dogs',
-                              slug: 'cats'
-                            },
-                            {
-                              description: 'what books are made of',
-                              slug: 'paper'
-                            }
-                          ]
-                    );
+          .then(({ body }) => {
+            const topics = body.topics;
+            expect(Array.isArray(topics)).toBe(true)
+            topics.forEach(topic => {
```

Comment:
As we're asserting in a forEach here, could we just add another expect to check the length of the array please?